romanisim/image.py

```diff
@@ -684,14 +692,17 @@ def simulate(metadata, objlist,
         )
         saturation = saturation_model.data[nborder:-nborder, nborder:-nborder]
     else:
-        read_noise = galsim.roman.read_noise
-        darkrate = galsim.roman.dark_current
-        dark = None
-        gain = None
-        flat = 1
-        inv_linearity = None
-        linearity = None
-        saturation = None
+        read_noise = parameters.reference_file_names["readnoise"] \
+            if parameters.reference_file_names["readnoise"] is not None else galsim.roman.read_noise
+        darkrate = (parameters.reference_file_names["dark"] / exptime_tau) \
```

Comment:
For the case where this is a constant, let's just have the constant be the dark current rather than the last value in the dark reference file divided by tau.

Reply:
Changed to a new keyword.